android/src/main/java/com/applicaster/plugin/televisionacademyplayer/network/NetworkAPI.kt

```diff
@@ -0,0 +1,17 @@
+package com.applicaster.plugin.televisionacademyplayer.network
+
+import com.applicaster.plugin.televisionacademyplayer.PlaybackURLResponse
+import retrofit2.Call
+import retrofit2.http.*
+
+interface NetworkAPI {
+
+    @GET("playback/{content_uid}?manifest_type=HLS")
+    @Headers("Accept: application/json")
+    fun getPlaybackURL(@Path("content_uid") forContentId : String, @Header("Authorization") token : String) : Call<PlaybackURLResponse>
+
+    @GET("fetchData?type=submissions&screen=videos&env=prod&isTVApp=false")
```

Comment:
Is it possible to expose this in the plugin configuration?

Reply:
done by migrate network to OKHttp